### ISSUE TITLE
Prevent maximized window snapoff (for use with the wobbly windows plugin)

### ISSUE THREAD

**Issue by RalfVB** [WayfireWM/wayfire#1733]
Compiz has a feature that prevents maximized windows from being moved when dragging the title bar (or from anywhere when dragging while pressing a defined key).

I loved this feature because it really showed off the 'wobbly windows' feature, since you could stretch and deform them in really impressive way, and they would snap back in a very satisfying manner when you stopped dragging, very much as if the windows were made of jelly.

Please consider adding this feature!

**Comment by soreau** [comment]
This feature already exists. In the move plugin, enable Snap Off option and set Snap Off Threshold to a high value.

**Comment by RalfVB** [comment]
Good call, this works indeed! I'm glad to have my jelly windows back :)

Is it possible to increase the amount of 'polygons' (I don't know what the term is here) so deformed windows look smoother?

**Comment by soreau** [comment]
> Good call, this works indeed! I'm glad to have my jelly windows back :)
> 
> Is it possible to increase the amount of 'polygons' (I don't know what the term is here) so deformed windows look smoother?

In wobbly plugin, grid resolution option.
